tools/README.md

```diff
@@ -0,0 +1,43 @@
+# Docker Image for Hugo v0.25.x Legacy JVM Driver Documentation Builds
+
+## Background
+
+The legacy documentation for MongoDB JVM-based drivers (Java Sync, Java RS, and Scala)
+were built with an older version of [``hugo``](https://github.com/gohugoio/hugo) and were
+hosted on GitHub Pages. Since the Java RS and Scala driver documentation hasn't been
+transitioned to MongoDB's Snooty Platform yet, we need to continue to publish the new
+documentation for each major and minor version release using the old tools.
+
+The documentation is built with the static site generator, ``hugo``. However, the
+pages were built long ago and did not keep up with the breaking changes in template syntax.
+Rather than attempt to update the syntax, the teams continued to use v0.25.x.
+
+When the company transitioned from Intel hardware to M1 Macs which use the "AArch64"
+architecture, the ``hugo`` binaries stopped working, even under emulation. While 
+the maintainers for ``hugo`` started building binaries for the architecture, the latest
+one for v0.25.x binary is not compatible. 
+
+However, it does run on a Linux environment on Docker. Therefore, we need to use
+Docker to build the legacy reference documentation for these drivers for every
+major and minor release until the team can transition the contents to the Snooty
+platform.
```

Comment:
Non-blocking nit-pick, but Snooty Platform was capitalized earlier in this section (currently line 8). I'd suggest either capping here or lowercasing there